exo-go/src/cmd/remove_template.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/os_helpers"
+	"github.com/Originate/exosphere/exo-go/src/template_helpers"
+	"github.com/Originate/exosphere/exo-go/src/util"
+	"github.com/spf13/cobra"
+)
+
+var removeTemplateCmd = &cobra.Command{
+	Use:   "remove-template",
+	Short: "Removes an existing service template from .exosphere",
+	Long: `Removes an existing service template from .exosphere
+This command must be called in the root directory of the application`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if util.PrintHelpIfNecessary(cmd, args) {
+			return
+		}
+		if len(args) != 1 {
+			fmt.Println("not enough arguments")
+      os.Exit(1)
```

Comment:
Please configure your editor to run `goimports` every time you save. This line here is indented with spaces, it should be tabs. @charlierudolph can help you set this up.